webservices/exceptions.py

```diff
@@ -1,3 +1,9 @@
+
+LINE_NUMBER_ERROR = """
+Invalid line_number detected. A valid line_number is using the following format:
+'FORM-LINENUMBER'.  Check the api doc for more info.
```

Comment:
I don't think `Check the api doc for more info.` adds value here, can you remove it?

Reply:
Maybe instead we could give them an example, like the docs do? From the docs: `For example an argument such as `F3X-16` would filter down to all entries from form `F3X` line number `16`.`

Reply:
Sure - that definitely adds value. There is a minor issue for the docs too: F3X-16 is only an example of Schedule_a, but it is used for both schedule_a and schedule_b. So it is a little confusing on this part.